OracleIdentityGovernance/dockerfiles/12.2.1.4.0/as.properties

```diff
@@ -0,0 +1,11 @@
+# # Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Property and configuration files do not need copyright or license headers. Please remove them from all `*.properties` files.

Reply:
done